Jetsnack/app/src/main/java/com/example/jetsnack/ui/MainActivity.kt

```diff
@@ -20,11 +20,13 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.enableEdgeToEdge
+import androidx.core.view.ViewCompat
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         enableEdgeToEdge()
         super.onCreate(savedInstanceState)
+        ViewCompat.setOnApplyWindowInsetsListener(window.decorView) { _, insets -> insets }
```

Comment:
This gets the inset and return it without doing anything. It seems to me that it doesn't do what the before/after screenshots do.
Is PR missing some commits?

Reply:
No, it isn't.
The MDC Catalog app does the same thing for edge-to-edge layout.
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/windowpreferences/WindowPreferencesManager.java#L80-L81
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/windowpreferences/WindowPreferencesManager.java#L41-L46